src/openfermion/chem/reduced_hamiltonian_test.py

```diff
@@ -1,10 +1,11 @@
 import os
 import numpy as np
-import openfermion as of
 from openfermion.config import DATA_DIRECTORY
-from openfermion.hamiltonians import MolecularData
-from openfermion.ops import InteractionOperator
-from openfermion.utils._reduced_hamiltonian import make_reduced_hamiltonian
+from openfermion.chem import MolecularData
+from openfermion.ops.representations import InteractionOperator
+from openfermion.chem.reduced_hamiltonian import make_reduced_hamiltonian
+import openfermion.linalg.sparse_tools as sps_tools
```

Comment:
Absolute imports work in 3.7.  This skirts a circular import issue.  It can likely be resolved now that we moved files around.  Why don't you try out being explicit about the functions to check if my fix of moving some circuit construction stuff out of sparse_tools works

Reply:
Ah ok. I'll give it a go and see if this works.